com_tc/site/views/content/tmpl/default.php

```diff
@@ -36,7 +36,7 @@ function validateform(form)
 			<br>
 			<div class="">
 				<?php
-						echo nl2br(htmlspecialchars($this->termsandconditions->content, ENT_COMPAT, 'UTF-8'));
+						echo nl2br($this->termsandconditions->content);
```

Comment:
why this change needed?

Reply:
If the T&C has created using TyniMCE editor then it showing <p> tag for text sentences on view page of T&C. but its working fine with MD editor if code contains htmlspecialchar method.

I tested it with removing htmlspecialchar() method for both editors also checked HTML and XSS attacks and its working fine.